cranelift/filetests/filetests/runtests/brif.clif

```diff
@@ -0,0 +1,178 @@
+test interpret
+test run
+target aarch64
+target s390x
+target x86_64
+target riscv64
+
+function %brif_value(i8) -> i64 {
```

Comment:
I'm not seeing any test here where the `then_block` and `else_block` have different numbers of block arguments. That could be good to exercise if it isn't done elsewhere.